### ISSUE TITLE
[Messenger] Move Transport/InMemoryTransport to Transport/InMemory/InMe…

### ISSUE THREAD

**Issue by nicolas-grekas** [symfony/symfony-docs#17573]
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#48614
| PR author(s) | @lyrixx
| Merged in    | 6.3

**Comment by lyrixx** [comment]
For the record:

```
>…e/dev/github.com/symfony/symfony-docs(translit-emoji) rg InMemoryTransport
messenger/custom-transport.rst
129:be implemented. For real implementations see :class:`Symfony\\Component\\Messenger\\Transport\\InMemoryTransport`

messenger.rst
1519:    use Symfony\Component\Messenger\Transport\InMemoryTransport;
1530:            /* @var InMemoryTransport $transport */
```

**Comment by javiereguiluz** [comment]
Fixed in #17574.
